analyses/snv-callers/scripts/01-calculate_vaf_tmb.R

```diff
@@ -97,29 +99,48 @@ option_list <- list(
     metavar = "character"
   ),
   make_option(
-    opt_str = "--vaf_filter", type = "numeric", default = 0,
+    opt_str = "--vaf_filter", type = "character", default = "0",
```

Comment:
Is there a reason you changed this to character? I prefer numeric when that is what you need.

Reply:
With the `optparse` options and other upstream things were inconsistent so this was my attempt to make it all the same. Let me take another look and figure out why I thought this was simpler when on face value it doesn't seem that way.